src/bp/ui-studio/src/web/components/Shared/Interface/index.tsx

```diff
@@ -95,24 +95,23 @@ export const SearchBar = (props: SearchBarProps) => {
 export const ItemList = (props: ItemListProps) => {
   return (
     <div className={style.itemList}>
-      <ul>
-        {props.items &&
-          props.items.map((item, idx) => (
-            <li key={idx}>
-              <div style={{ width: '80%' }} onClick={() => props.onElementClicked && props.onElementClicked(item)}>
-                {item.label}
-              </div>
-              <div style={{ marginLeft: 'auto ' }}>
-                {props.actions &&
-                  props.actions.map(action => (
-                    <Tooltip key={idx + action.tooltip} content={action.tooltip} position={Position.RIGHT}>
-                      <Icon icon={action.icon} onClick={() => action.onClick && action.onClick(item)} />
-                    </Tooltip>
-                  ))}
-              </div>
-            </li>
-          ))}
-      </ul>
+      {props.items &&
+        props.items.map((item, idx) => (
```

Comment:
Please try not to use idx as key, it will result in poor performance and sometimes weird behaviour because your key is not uniq to the element. Check this out if you need another source of explanation : https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c